### PR TITLE
feat(tabselector.ts): added a selector for selected tab

### PR DESCRIPTION
**Pull request by FelixBlaisThon**
### Proposed Changes

add selector for selected tab

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/react-vapor/src/components/tab/tests/TabSelectors.spec.tsx [hunk 1]
```diff
@@ -47,5 +47,16 @@ describe('TabSelector', () => {
             const expectedTabGroup = tabGroupState.tabs[0];
             expect(tabGroup).toEqual(expectedTabGroup);
         });
+        it('should return true if a tab is selected, false if not', () => {
```
Comment (gdostie) [inline packages/react-vapor/src/components/tab/tests/TabSelectors.spec.tsx:50]: It should be outside the describe starting on [line 8.](https://github.com/coveo/react-vapor/pull/1622/files#diff-1687ca765e3cd4c51dfed88d055bb27eR8)

Comment (gdostie) [inline packages/react-vapor/src/components/tab/tests/TabSelectors.spec.tsx:50]: The case where the tab with the specified id doesn't exist in the state is not covered (it should return false).

Comment (FelixBlaisThon) [reply packages/react-vapor/src/components/tab/tests/TabSelectors.spec.tsx:50]: woops

File: packages/react-vapor/src/components/tab/tests/TabSelectors.spec.tsx [hunk 2]
```diff
@@ -48,4 +48,28 @@ describe('TabSelector', () => {
...
+
+        it('should return false if the tab does not exist in the state', () => {
+            const tabGroupState = {
+                id: DEFAULT_GROUP_ID,
+                tabs: [
+                    {id: 'coulili-zazou', isSelected: true},
+                    {id: 'gros-boudesse', isSelected: false},
+                ],
+            };
+            expect(TabSelectors.getTabSelected('who-am-i').resultFunc(tabGroupState)).toBe(undefined);
```
Comment (gdostie) [inline packages/react-vapor/src/components/tab/tests/TabSelectors.spec.tsx:72]: It should return `false` meaning the tab isn't selected.
```suggestion
            expect(TabSelectors.getTabSelected('who-am-i').resultFunc(tabGroupState)).toBe(false);
```

**Pull request merged**